apps/medon-fe/src/components/Notification/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useMemo, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { INotification, NotificationType } from 'components/Notification/types';
+import {
+  formatTimeDifference,
+  roles,
+  routes,
+  timerTimeout,
+} from 'utils/constants';
+import { ReactComponent as InfoIcon } from 'assets/images/dashboard/Info.svg';
+import { Content, Wrapper } from 'components/Notification/styles';
+
+export function Notification({
+  user,
+  appointment,
+  timerType,
+  renderTitle,
+  type,
+}: INotification) {
+  const navigate = useNavigate();
+  const { t } = useTranslation();
+
+  const [timer, setTimer] = useState<string>(
+    formatTimeDifference(appointment.startTime, timerType)
+  );
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setTimer(formatTimeDifference(appointment?.startTime, timerType));
+    }, timerTimeout);
+    return () => {
+      clearInterval(interval);
+    };
+  }, [user, appointment]);
+
+  const doctor = useMemo(
+    () =>
+      user.role === roles.local
+        ? appointment?.remoteDoctor?.lastName
+        : appointment?.localDoctor?.lastName,
+    [user, appointment]
+  );
+
+  const renderingCondition =
```

Comment:
no one will understand what you try to do here, or change variable name to one that will explain it, or add comments

Reply:
fixed